src/groupInfo/datastore/groups.js

```diff
@@ -41,6 +41,7 @@ export default {
       return Object.values(state.entries).map(getters.enrich)
     },
     mine: (state, getters) => getters.all.filter(e => isMyGroup(e)).sort(applicationsFirstThenSortByName),
+    isMemberGroups: (state, getters) => getters.all.filter(e => isMemberGroup(e)).sort(applicationsFirstThenSortByName),
```

Comment:
Also minor.

I find the name `isMemberGroups` confusing, as normally something starting with `is` will return a boolean value (e.g. the `isMemberGroup` function).

I would probably call it `mineWithoutApplications` ... or maybe call it `mine` and rename the existing `mine` to be `mineWithApplications`.

Reply:
I went for `mine` and `mineWithApplications`. Hopefully I managed to change it wherever it is referenced. Nothing seems to be broken